web/src/components/ContactItem.tsx

```diff
@@ -20,17 +22,29 @@ const StyledLink = styled(Link)`
   ${helpers.adaptiveFontSize};
 `
 
+
+const StyledExternalLinkIcon = styled.img`
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  object-fit: contain;
+  margin-left: 8px; // Добавляем отступ между текстом и иконкой
```

Comment:
🙃 Firstly, we try to keep the code in English as much as possible, secondly, Google tells me this says "Add spacing between the text and the icon" which is what the code says, so please delete the comment